readthedocs/projects/tasks/utils.py

```diff
@@ -61,16 +61,22 @@ def clean_project_resources(project, version=None):
     - Imported files.
 
     :param version: Version instance. If isn't given,
-                    all resources of `project` will be deleted.
+     all resources of `project` will be deleted.
```

Comment:
> :param version: Version instance. If isn't given,
>     all resources of `project` will be deleted.

We need to adapt it to mention "If neither of version and version_slug are given,"